schema.go

```diff
@@ -1007,7 +1007,7 @@ var Schema = schema.NewTypedScopeSchema[*Config](
 				nil,
 			),
 			"cacert": schema.NewPropertySchema(
-				schema.NewStringSchema(schema.IntPointer(1), nil, regexp.MustCompile(`^\s*-----BEGIN CERTIFICATE-----(.|\n)+-----END CERTIFICATE-----\s*$`)),
+				schema.NewStringSchema(schema.IntPointer(0), nil, regexp.MustCompile(`^|\s*-----BEGIN CERTIFICATE-----(.|\n)+-----END CERTIFICATE-----\s*$`)),
```

Comment:
Does an insecure cluster require a blank string, or should we make this field optional?
In the future, this would be a good candidate for the plan to add a RequiredIfTrue schema property.

Reply:
You're right, not 100% if it was nil or empty string... I left it only to change less possible, tomorrow I'll double check and apply the change. Glad to hear that you guys don't have any issue with this change, it's really important for us 🙏.